apps/penxle.com/src/lib/tiptap/node-views/access-barrier/Component.svelte

```diff
@@ -43,6 +50,11 @@
     <div class="flex grow flex-col">
       <div class="font-bold">결제 상자</div>
       <div class="text-sm text-gray-50">이 상자 아래로는 결제를 해야 읽을 수 있어요.</div>
+
+      <div class="flex gap-2 items-center mt-2">
+        <TextInput class="bg-gray-10 px-3 py-2" value={node.attrs.price} on:input={handlePriceChange} />
```

Comment:
```suggestion
        <TextInput class="bg-gray-10 px-3 py-2" value={node.attrs.price} on:input={handlePriceChange} inputmode="numeric" />
```

P3: 숫자 형식의 텍스트라는 걸 알려주기 위해 힌트를 주는게 좋다고 생각합니다!

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode#numeric